docs/configuration/general/cookies.rst

```diff
@@ -0,0 +1,122 @@
+.. _configuration_general_cookies:
+
+Privacy and cookies
+===================
+
+By default, Open Forms only uses (first party) `strictly necessary cookies`_ 
+(although sometimes confusingly called "functionality cookies"). These cookies 
+are needed to make Open Forms work: The browser should remembers which form 
+someone is currently doing and whether you are authenticated in or not.
+
+For this purpose, Open Forms does not require consent according to the `GDPR`_
+(although local laws may differ in this aspect). You are encouraged to at least 
+add a link to your privacy policy to explain why these cookies are required.
+
+.. _`strictly necessary cookies`: https://gdpr.eu/cookies/
+.. _`GDPR`: https://gdpr.eu/
+
+
+Pivacy related settings
+-----------------------
+
+There are 3 related settings to privacy:
+
+**Link to privacy policy**
+
+When you have an existing privacy policy, you can link to the privacy policy 
+page or document from Open Forms. When enabled, this link is shown in the 
+footer and can also be shown when asking for permission before submitting a 
+form (see below).
+
+**Ask permission to process user data before submitting a form**
+
+If you created a form that asks for personal data, it's a good idea to enable
+this option to make the user aware they are sending personal data to your 
+organisation. When enabled, a checkbox is shown on the summary page that users
+need to check before they can submit the form. The message next to the checkbox
+is configurable and can optionally contain the link to the configured privacy 
+policy by using ``{% privacy_policy %}`` in the label.
+
+.. image:: _assets/agree_to_process_data.png
+
+**Ask for cookie consent**
+
+This is also known as the cookie bar or cookie notice. You can create an 
+overview of your cookies and ask for the users consent when they first arrive
+at a form. All non-required cookies are rejected until the user explicitly
+allows these cookies. Required cookies cannot be rejected.
+
+Although this is optional, if you enable any of the settings 
+for :ref:`analytics <configuration_general_analytics>` you will need to 
+configure some cookie settings first.
+
+.. image:: _assets/cookie_notice.png
+
+.. image:: _assets/manage_cookies.png
+
+
+Configure cookies
```

Comment:
I think this is missing the step in the global configuration to specify which cookie group is used for analytics.

Reply:
I tackled that in the analytics part.

Reply:
Nope, I did not. Will add.

Reply:
Doh, it was tackled below in the section **Configure privacy**